apps/mobile/src/components/FadedInput.tsx

```diff
@@ -4,14 +4,22 @@ import { TextInput, TextInputProps, View } from 'react-native';
 
 import colors from '~/shared/theme/colors';
 
-type Props = TextInputProps & { icon?: ReactNode };
+type Props = TextInputProps & { icon?: ReactNode; editable?: boolean; textClassName?: string };
```

Comment:
If Im not mistaken, the `editable` props already exist inside the `TextInputProps`